lblod/ember-rdfa-editor-lblod-plugins
# Rdfa/actual structure compat

### Overview
Some backwards compatibility fixes for article structure plugin nodetypes.

##### connected issues and PRs:
Requires https://github.com/lblod/ember-rdfa-editor/pull/1128

### Setup
`npm i` should be enough as it's referencing the git commit it needs.

### How to test/reproduce
Paste different old html containing article structures into the 'custom html' input in the editor. HTML can be taken from e.g. qa GN. Check that all the relationships are shown and clicking on outgoing relationships and backlinks works. Refresh and make sure it still works.

### Challenges/uncertainties
Aside from dependency problems, a difficulty was not knowing when something was already broken just being hidden by prosemirror's default behaviour. It would also be very valuable to have a collection of old documents that we can test with.

### Checks PR readiness
